code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -419,8 +419,7 @@
 			return
 		build_step++
 		to_chat(user, "<span class='notice'>You complete the Securitron! Beep boop.</span>")
-		var/mob/living/simple_animal/bot/secbot/S = new /mob/living/simple_animal/bot/secbot
-		S.loc = get_turf(src)
+		var/mob/living/simple_animal/bot/secbot/S = new /mob/living/simple_animal/bot/secbot(get_turf(src))
```

Comment:
Can't, doesn't return the atom. That's more for spawning multiple things to the turf for admins